src/Runtime/jni/CMakeLists.txt

```diff
@@ -11,9 +11,11 @@ if(Java_Development_FOUND AND JNI_FOUND)
   # Target for Java runtime jar
   add_jar(javaruntime
           src/com/ibm/onnxmlir/OMModel.java
-	  src/com/ibm/onnxmlir/OMTensorList.java
-          src/com/ibm/onnxmlir/OMTensor.java
-          OUTPUT_DIR ${CMAKE_LIBRARY_OUTPUT_DIRECTORY})
```

Comment:
Why not just change this to `${ONNX_MLIR_LIBRARY_PATH}`?

Reply:
Because ${ONNX_MLIR_LIBRARY_PATH} right now has the generator expression $<CONFIG>, this fails. Other add_* functions like add_library support generator expressions, but add_jar does not. It tries to make sure that the directory exists, so with multi-config generators, it tries to create a path containing $<CONFIG> before it has been resolved to each of the configurations and fails.

Reply:
Thanks for the clarification. So it would be nice to have these as the comments about the copy command below.